Dinkers/The-Curation
# #22 update city select

Closing #22

The City Select component on the Home screen now appears as a single line rather than a massive image.

## Summary
- Updated UI for the CitySelect component
- Added font-awesome for a nice little drop-down arrow (wasn't worth it) 
- Added a Button component as I plan to re-use this for the Place Type Select component
- Fixed UI bug with CitySelect modal cutting off content with a chunk of whitespace
- Converted all components to use arrow functions because it's 2021
- Re-added `package-lock.json`, that's where the 21k line changes are coming from

## Testing steps
- Pull down `enhancement-22-update-city-select`
- npm install
- `npm run` it from client dir
-  Marvel at, then click the button
- It should show the City Select Modal as usual, but with more content and less white-space
- You should be able to dismiss the modal/select a city, and everything works as expected

## Screeny
![Screen Shot 2021-06-22 at 3 21 19 PM](https://user-images.githubusercontent.com/3612297/123015219-b8840f80-d37c-11eb-9c20-275e230fca7f.png)
